### PR TITLE
Add myself

### PR DESCRIPTION
**Pull request by eitan-weinstein**

### CODE REVIEW

File: community/people/ew/docker
Comment (gonuke) [inline community/people/ew/docker:1]: I don't think this file is necessary.

File: community/people/ew/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+phone: "612-245-1556"
+city_state_zip: "Madison, WI 53706"
+services:
+  github: "eitan-weinstein"
+  facebook: "eitan.weinstein.98"
+  linkedin: "eitan-weinstein-792101224"
+ 
+ ---
+ 
+ "Eitan Weinstein is an undergraduate student at the University of Pennsylvania
```
Comment (gonuke) [inline community/people/ew/index.md:16]: I think you probably want to remove the quotation marks here.

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks for this contribution @eitan-weinstein - I have a couple of comments on the files here.  I think you'll also need to add yourself to the file `_data/people.yml` in the section that includes other undergraduates.

File: community/people/ew/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+image: "EitanHeadshot.jpg"
+email: "esweinstein2@wisc.edu"
+phone: "612-245-1556"
+city_state_zip: "Madison, WI 53706"
+services:
+  github: "eitan-weinstein"
+  facebook: "eitan.weinstein.98"
+  linkedin: "eitan-weinstein-792101224"
+ 
+ ---
```
Comment (gonuke) [inline community/people/ew/index.md:14]: I also discovered an extra space at the beginning of this line that will be a problem! Subtle bug

**Comment by eitan-weinstein** [comment]
> Thanks for this contribution @eitan-weinstein - I have a couple of comments on the files here. I think you'll also need to add yourself to the file `_data/people.yml` in the section that includes other undergraduates.

It's possible I did this wrong, but when I went back in to add myself into the people.yml file, it looked like my initial addition was successful and I was seeing my entry there. I put myself right after Gerson's entry.

**Comment by eitan-weinstein** [comment]
> > Thanks for this contribution @eitan-weinstein - I have a couple of comments on the files here. I think you'll also need to add yourself to the file `_data/people.yml` in the section that includes other undergraduates.
> 
> It's possible I did this wrong, but when I went back in to add myself into the people.yml file, it looked like my initial addition was successful and I was seeing my entry there. I put myself right after Gerson's entry.

Nevermind, I think I may have figured out the issue. I will submit another PR in a moment.

File: community/people/ew/EitanHeadshot.jpg
Comment (gonuke) [inline community/people/ew/EitanHeadshot.jpg:1]: This looks a lot better, but one last issue - the photos are forced to be square on each person's page.  Can you crop this to a roughly square size?

**Review by gonuke — CHANGES_REQUESTED** [review]
I think you managed to clear up the prior issues. Just one last request for a square (or nearly square) cropped image.  (Maybe we need to add that guidance to the how-to page as well.)

**Comment by gonuke** [comment]
This looks good now @eitan-weinstein! merging now

**Review by gonuke — APPROVED** [review]

**Pull request merged**